raft/src/main/java/org/apache/kafka/snapshot/SnapshotWriter.java

```diff
@@ -107,7 +107,7 @@ public boolean isFrozen() {
     public void append(List<T> records) throws IOException {
         if (snapshot.isFrozen()) {
             String message = String.format(
-                "Append not supported. Snapshot is already frozen: id = {}.",
+                "Append not supported. Snapshot is already frozen: id = {%s}.",
```

Comment:
Is ```'%s'``` more suitable (if you want to highlight the object string) ?

Reply:
oh, I didn't notice that it's a object.